pkg/controller/controller.go

```diff
@@ -61,28 +62,37 @@ func New(cfg Config) *Controller {
 	}
 }
 
-func (c *Controller) handleClusterEvent(event *Event) error {
+// handleClusterEvent returns ignored true if cluster isn't managed by this instance.
+func (c *Controller) handleClusterEvent(event *Event) (ignored bool, err error) {
```

Comment:
Sorry I wasn't clear before.
Prefer ALL returns to be explicit about return values.
```
handleClusterEvent(event *Event) (bool, error) {
  ...
  return false, nil
  ...
  return false, fmt.Errorf(...)

```

Reply:
hehe no problem :)